src/utils/request.ts

```diff
@@ -149,3 +149,20 @@ export function getRequestURL(
   const path = getRequestPath(event);
   return new URL(path, `${protocol}://${host}`);
 }
+
+export function getRequestIP(
+  event: H3Event,
+  opts: { xForwardedFor?: boolean } = {},
+) {
+  const nonProxyIp = (event.node.req.connection as any).remoteAddress;
+
+  if (!opts.xForwardedFor) {
```

Comment:
I think we should make it opt-in strictly to avoid issues.

Reply:
This is opt-in isn't it